scripts/test-suite.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-TUPELO_VERSION=g4-token-proofs docker-compose -f docker-compose.yml -f docker-compose.test.yml up --force-recreate --abort-on-container-exit
```

Comment:
my vote is just take out `TUPELO_VERSION` since the docker-compose files default to master anyways, and that way you can do `TUPELO_VERSION=branch npm run test-suite` - will PR that proposal against this

Reply:
https://github.com/quorumcontrol/tupelo-wasm-sdk/compare/upgrade/tupelo-go-sdk...upgrade/tupelo-go-sdk-bw?expand=1